### ISSUE TITLE
Typo of Chapter 7

### ISSUE THREAD

**Issue by Magentaize** [rasbt/python-machine-learning-book-2nd-edition#103]
Since I am reading a photocopy in China, I am sorry that I don’t know the typo page.
The typo is located in the "Note" of section "Implementing a simple majority vote classifier".
![image](https://user-images.githubusercontent.com/11793304/52627003-6b135000-2eef-11e9-834b-0b3d9a0bf5f5.png)
this set should be {1,2,3} likes the shown below.

**Comment by rasbt** [comment]
I agree, thanks for noting that. It will be fixed in the 3rd edition.
